Makefile

```diff
@@ -239,6 +239,9 @@ build/dmadata_table_spec.h: build/$(SPEC)
 
 build/src/boot/z_std_dma.o: build/dmadata_table_spec.h
 build/src/dmadata/dmadata.o: build/dmadata_table_spec.h
+build/src/code/z_actor_dlftbls.o: include/tables/actor_table.h
+build/src/code/z_scene.o: include/tables/object_table.h
+build/src/code/z_effect_soft_sprite_dlftbls.o: include/tables/effect_ss_table.h
```

Comment:
I really don't think we should start manually declaring dependencies like this.
Imo these files should be treated like any other include, ie. if you modify them you have to touch the corresponding files or do a a full build for the changes to apply, at least until we restructure headers and add automatic include dependencies to the makefile. (Hopefully this should happen sooner rather than later, tho maybe not before retail support)

Reply:
I understand the concerns of manually defined dependencies, but these tables are such a thing that modding will definitely touch that I would really like to have this in the repo now instead of in several months

These tables aren't typical includes either